packages/meditrak-server/src/routes/postChanges.js

```diff
@@ -37,6 +44,7 @@ export async function postChanges(req, res) {
     if (!ACTION_HANDLERS[action]) {
       throw new ValidationError(`${action} is not a supported change action`);
     }
+    await PAYLOAD_TRANSLATORS[action](models, payload);
```

Comment:
This was my solution of how to submit survey responses by code/email/etc. Basically we take in the payload, and we `translate` the unsupported fields (`question_code`, `user_email`, etc.) into the expected fields (`question_id`, `user_id`, `assessor_name`, etc.).

I don't particularly like this solution in the long term, and I think are more thought out refactor of how these endpoints work is called for... Perhaps we associate requests with a user, and allow for a more generic id/code referencing system?

However, for now I think this is lower risk, and quicker and easier to implement...

Happy to get push back on this whole implementation if people find it too hacky 👍

Reply:
I'm actually ok with this implementation - I don't think we need to have a different mapping for incoming data per client, given that they're pushing to us. But I don't think it's quite in the right place - I'd like `/changes` to be restricted to supporting syncing clients (so far just the data collection app).